### ISSUE TITLE
Are you open for a PR that allows an extra module to be enabled via an ENV var (default off)?

### ISSUE THREAD

**Issue by jackkinsella** [heroku/heroku-buildpack-nginx#111]
TLDR: If I created a PR that made the `--with-http_stub_status_module` module possible to include via an ENV var (e.g. `NGINX_WITH_X`, would you accept this?

Background:

We want to log the active number of worker connections in nginx in order to proactively keep on top of potential worker resource starvation. The `--with-http_stub_status_module` module makes the `$number_connections` variable available.

The suggested solution of going with a fork is not worth the risk of potentially missing critical upstream changes to the buildpack.

Obviously even better would be if this small module were included by default in the BuildPack (it's default in certain installations, e.g. via Homebrew on macos). But I understand if you would prefer not to add default modules.

**Comment by rbrisita** [comment]
Or add the ability to read in `scripts/build_nginx` that contains added build configuration options.

**Comment by rbrisita** [comment]
Easiest implementation PR found here: https://github.com/heroku/heroku-buildpack-php/issues/598
